### PR TITLE
[stable/MM-TE] add extra initcontainers and envvars

### PR DESCRIPTION
**Pull request by cpanato**
#### What this PR does / why we need it:
Add the ability to add extra initContainers and environments variables to the deployment

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mattermost-team-edition/templates/deployment.yaml [hunk 1]
```diff
@@ -27,17 +27,24 @@ spec:
...
+      {{- if .Values.extraInitContainers }}
+{{ tpl .Values.extraInitContainers . | indent 6 }}
+      {{- end }}
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.imagePullPolicy }}
+        env:
+          {{- if .Values.extraEnvVars }}
+{{ toYaml .Values.extraEnvVars | indent 12 }}
```
Comment (davidkarlsen) [inline stable/mattermost-team-edition/templates/deployment.yaml]: 10 or 12?

Comment (cpanato) [reply stable/mattermost-team-edition/templates/deployment.yaml]: I already tested with those values and worked. 
I can add a CI entry to test this out :)

Comment (cpanato) [reply stable/mattermost-team-edition/templates/deployment.yaml]: but it 10 to match the others :) fixing

Comment (cpanato) [reply stable/mattermost-team-edition/templates/deployment.yaml]: done

**Pull request merged**